docs/docs/cmd/adaptivecard/adaptivecard-send.md

```diff
@@ -43,6 +43,8 @@ The predefined card is automatically adjusted based on which options have been s
 
 If your custom card is a card template (card with placeholders like `${title}`), you can fill it with data either by specifying the complete data object using the `cardData` option, or by passing any number of arbitrary options that will be mapped onto the card. The arbitrary properties should not match any of the global options like `output`, `query`, `debug`, etc. Data options like `title`, `description`, `imageUrl` and `actionUrl` will be mapped onto the card as well.
 
+Using different shells, we see some differences in how quotes behave, which is why the following examples will work on Bash/zsh/fish (sh) on macOS. See below for examples that will work on Windows Powershell
```

Comment:
This statement applies to all samples across all of our docs. Rather than mentioning it here specifically, let's start a new discussion to see how we can approach it consistently across all commands. Thank you for bringing it up 👍